LibreNMS/Data/Store/InfluxDBv2.php

```diff
@@ -31,11 +31,74 @@
 use InfluxDB2\Client;
 use InfluxDB2\Model\WritePrecision;
 use InfluxDB2\Point;
+use InfluxDB2\WriteType;
 use LibreNMS\Config;
 use Log;
 
 class InfluxDBv2 extends BaseDatastore
 {
+    private $client;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        try {
+            $host = Config::get('influxdbv2.host', 'localhost');
+            $transport = Config::get('influxdbv2.transport', 'http');
+            $port = Config::get('influxdbv2.port', 8086);
+            $bucket = Config::get('influxdbv2.bucket', 'librenms');
+            $organization = Config::get('influxdbv2.organization', '');
+            $allow_redirects = Config::get('influxdbv2.allow_redirects', true);
+            $token = Config::get('influxdbv2.token', '');
+            $debug = Config::get('influxdbv2.debug', false);
+            $log_file = Config::get('influxdbv2.log_file', Config::get('log_file'));
```

Comment:
Did you try this with writing to the normal librenms.log? Did it cause any issues?

Reply:
I did try, didn't got any issue and was really useful during a debug, this would only start to log while the "debug" option is enabled.